DOCUMENTATION.md

```diff
@@ -1,5 +1,24 @@
 # Documentation
 
+## Table of contents
+
+- [Flatpak permissions](#flatpak-permissions)
+	- [Permissions](#permissions)
+	- [Share](#share)
+	- [Socket](#socket)
+	- [Device](#device)
+	- [Allow](#allow)
+	- [Filesystem](#filesystem)
+	- [Persistent](#persistent)
+	- [Environment](#environment)
+	- [System Bus](#system-bus)
+	- [Session Bus](#system-bus)
+- [Tips and Tricks](#tips-and-tricks)
+	- [Manually reset Flatseal permissions](#manually-reset-flatseal-permissions)
+	- [Enable custom installations](#enable-custom-installations)
+	- [Flatpak 1.7.1 or newer](#flatpak-171-or-newer)
+	- [All versions](#all-versions)
+
 ## Flatseal permissions
 
 This section indicates the permissions of Flatpak with better integration with Flatseal. It is based on the official [Sandbox Permissions Reference](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html).
```

Comment:
This whole introduction (and the following Permissions sub section) should be simplified to something like:

## Permissions

This is the list of permissions supported by Flatseal. These descriptions are based on Flatpak's [official documentation](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html) and extended with examples and references to make it easier for newcomers to understand.

Both Flatseal and `flatpak override` CLI, use the same overrides backend e.g. both write to ~/.local/share/flatpak/overrides. A `flatpak override` equivalent column is added for those experienced with this CLI tool.